api/azure/webhook.go

```diff
@@ -70,7 +70,7 @@ func handleCheckRunEvent(log shared.Logger, azureAPI API, aeAPI shared.AppEngine
 		labels := mapset.NewSet()
 		sender := event.GetSender().GetLogin()
 		if sender != "" {
-			labels.Add(sender)
+			labels.Add("user:" + sender)
```

Comment:
Optional: it would be good to test the Azure webhook, too.

Reply:
Will do in a follow-up cleanup.